jerry-core/jerry.c

```diff
@@ -1774,11 +1778,18 @@ jerry_parse (const jerry_api_char_t *source_p, /**< script source */
  * @return completion status
  */
 jerry_completion_code_t
-jerry_run (jerry_api_object_t **error_obj_p)
+jerry_run (jerry_api_value_t *error_value_p)
```

Comment:
Since we expect the caller to release the out param `error_value_p`, this should be mentioned in the doc comment. Elsewhere in this file we have notes like:
```c
  * Note:
  *      returned value should be freed with jerry_api_release_value
  *      just when the value becomes unnecessary.
```

BTW, now that we are touching this function, it would be a good time to add a doc comment to the parameter as well.